metricbeat/beater/event.go

```diff
@@ -38,7 +34,7 @@ func (b eventBuilder) build() (common.MapStr, error) {
 
 	// Get and remove meta fields from the event created by the MetricSet.
 	indexName := getIndex(event, "")
-	typeName := getType(event, defaultType)
+	typeName := getType(event, b.moduleName+"."+b.metricSetName)
```

Comment:
This currently applies to `type` and `_type`. I think if we are going to change this, we should find a better name and not use `type`.